static/src/js/components/DownloadHistory/DownloadHistory.jsx

```diff
@@ -16,144 +16,131 @@ import PortalLinkContainer from '../../containers/PortalLinkContainer/PortalLink
-                            <PortalLinkContainer
-                              portalId={portalId}
-                              to={
-                                {
-                                  pathname: `/downloads/${id}`,
-                                  search: stringify({ ee: earthdataEnvironment === deployedEnvironment() ? '' : earthdataEnvironment })
-                                }
+      {
+        retrievalHistoryLoaded && (
+          retrievalHistory.length > 0 ? (
+            <Table className="download-history-table">
+              <thead>
+                <tr>
+                  <th className="download-history-table__contents-heading">Contents</th>
+                  <th className="download-history-table__created-heading">Created</th>
+                  <th className="download-history-table__actions-heading">Actions</th>
+                </tr>
+              </thead>
+              <tbody>
+                {
+                  retrievalHistory.map((retrieval) => {
+                    const {
+                      id,
+                      created_at: createdAt,
+                      jsondata,
+                      collections
+                    } = retrieval
+
+                    const { portal_id: portalId } = jsondata
+
+                    return (
+                      <tr key={id}>
+                        <td>
+                          <PortalLinkContainer
+                            portalId={portalId}
+                            to={
+                              {
+                                pathname: `/downloads/${id}`,
+                                search: stringify({ ee: earthdataEnvironment === deployedEnvironment() ? '' : earthdataEnvironment })
                               }
-                            >
-                              {this.retrievalDescription(collections)}
-                            </PortalLinkContainer>
-                          </td>
-                          <td className="download-history-table__ago">
-                            <TimeAgo date={createdAt} />
-                          </td>
-                          <td className="download-history-table__actions">
-                            <Button
-                              className="download-history__button download-history__button--remove"
-                              onClick={() => this.onHandleRemove(id)}
-                              variant="naked"
-                              icon={XCircled}
-                              label="Delete Download"
-                            />
-                          </td>
-                        </tr>
-                      )
-                    })
-                  }
-                </tbody>
-              </Table>
-            ) : (
-              <p>No download history to display.</p>
-            )
+                            }
+                          >
+                            {retrievalDescription(collections)}
```

Comment:
Im tempted to say we should memoize the result of this against the retrievalHistory so we dont have to recalculate it on every render.